sentry-ruby/lib/sentry/span.rb

```diff
@@ -90,11 +90,11 @@ def initialize(
 
     # Finishes the span by adding a timestamp.
     # @return [self]
-    def finish
+    def finish(end_timestamp: nil)
       # already finished
       return if @timestamp
```

Comment:
This isn't mentioned in the spec so I want to double check it: we only allow the first override but not the second time or later?
If that's true, can we add a test case for it?

Reply:
ah no, it should override either way, good catch!